Разметка

VovaLovalova opened the pull request:
---
:mortar_board: [Разметка](https://up.htmlacademy.ru/adaptive/26/user/2070387/tasks/4)

VovaLovalova committed:
ДЗ Разметка 3х страниц

DoberBober commented on source/catalog.html:
```diff
@@ -0,0 +1,263 @@
+      <a class="main-header__logo-picture" href="#">
+        <img src="./img/logo-mobile.svg" width="34px" height="38px" alt="cat energy">
+      </a>
+      <a class="main-header__logo-text" href="#">
```
Не нужно делить картинку и надпись. Вставьединой свгшкой.

DoberBober commented on source/catalog.html:
```diff
@@ -0,0 +1,263 @@
+      <section class="catalog">
+        <div class="catalog-wrapper">
+          <div class="catalog__card card">
+            <h3 class="card__title">Cat Energy PRO 500 г</h3>
```
Внутрь заголовка вставь ссылку, чтобы можно было перейти в карточку.

DoberBober commented on source/catalog.html:
```diff
@@ -0,0 +1,263 @@
+        <div class="catalog-wrapper">
+          <div class="catalog__card card">
+            <h3 class="card__title">Cat Energy PRO 500 г</h3>
+            <img class="card__image" src="./img/catalog/chiken-energy-500.png" width="68px" height="86px" alt="Cat Energy PRO курица 500 г">
```
Картинку тоже оберни в ссылку.

DoberBober commented on source/catalog.html:
```diff
@@ -0,0 +1,263 @@
+                <td class="product-description__value">700 Р</td>
+              </tr>
+            </table>
+            <button class="card__button button" type="button">Заказать</button>
```
Ссылка.

DoberBober commented on source/catalog.html:
```diff
@@ -0,0 +1,263 @@
+          <div class="catalog__card card-addition">
+            <h3 class="card-addition__title">Показать еще 100500 товаров</h3>
+            <p class="card-addition__description">На самом деле вкусов гораздо больше!</p>
+            <button class="card-addition__button button button--disebled" type="button">показать все</button>
```
Тоже ссылка.

DoberBober commented on source/catalog.html:
```diff
@@ -0,0 +1,263 @@
+                  <td class="additional-product-description__value">200 Р</td>
+                </tr>
+              </table>
+              <button class="additional-card__button button" type="button">Заказать</button>
```
И тут ссылка.

DoberBober commented on source/catalog.html:
```diff
@@ -0,0 +1,263 @@
+            <h2 class="additional-goods__title">Дополнительные товары</h2>
+            <div class="additional-goods_card additional-card">
+              <h3 class="additional-card__title">Сахарозаменитель</h3>
+              <table class="additional-card__list additional-product-description">
```
Тут спорный момент, но, как мне кажется, не очень хорошо таблица подходит. Выглядит, да, как таблица, но данные не табличные. Лучше сделай дивами.

DoberBober commented on source/index.html:
```diff
@@ -1,12 +1,155 @@
+          <p class="example__description">
+            Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день.
+          </p>
+          <fieldset class="example__legend example__legend--weight">
```
Филдсет как-то не в тему тут. Просто дивами.

DoberBober commented on source/index.html:
```diff
@@ -1,12 +1,155 @@
+          <p class="example__expense">Затраты на питание: 15 000 руб.</p>
+            <img class="example__cat-fat" src="./img/cat-fat.png" width="280px" height="232px" alt="Толстый кот">
+            <p class="example__radio-name example__radio-name--fat">
+              <input class="example__button example__button--before" type="radio" checked>
```
Было/стало - кнопки. Между ними input range.

DoberBober reviewed (changes requested):
С БЭМом и классами прям отлично, молодец. Есть небольшие правки по тегам и структуре, но это мелочи. Поправь и скидывай на повторную проверку.

VovaLovalova committed:
Исправил ДЗ по разметке

Не совсем понял как объединить лого из текста и картинки. Подумаю и исправлю на следующем этапе, если это обязательное условие

VovaLovalova committed:
Исправил лого

DoberBober reviewed (approved):
lgtm.
